packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -39,7 +44,7 @@ type TGetCFDAccountKey = TGetAccount & {
 // *
 // sub_account_type financial_stp only happens in "financial" market_type
 export const getCFDAccountKey = ({ market_type, sub_account_type, platform, shortcode }: TGetCFDAccountKey) => {
```

Comment:
Need to test out all the scenarios in which this function is being used to make sure your changes are not breaking anything because of line no. 61 and 81

Reply:
Already checked some of the cases of this function related to Financial and Synthetic.